staff/maryam-malek/pre-course/pasa-palabra/main.js

```diff
@@ -0,0 +1,105 @@
+    { letter: "p", answer: "protoss", status: 0, question: "CON LA P. Raza ancestral tecnológicamente avanzada que se caracteriza por sus grandes poderes psíonicos del videojuego StarCraft"},
+    { letter: "q", answer: "queso", status: 0, question: "CON LA Q. Producto obtenido por la maduración de la cuajada de la leche"},
+    { letter: "r", answer: "raton", status: 0, question: "CON LA R. Roedor"},
+    { letter: "s", answer: "stackoverflow", status: 0, question: "CON LA S. Comunidad salvadora de todo desarrollador informático"},
+    { letter: "t", answer: "terminator", status: 0, question: "CON LA T. Película del director James Cameron que consolidó a Arnold Schwarzenegger como actor en 1984"},
+    { letter: "u", answer: "unamuno", status: 0, question: "CON LA U. Escritor y filósofo español de la generación del 98 autor del libro 'Niebla' en 1914"},
+    { letter: "v", answer: "vikingos", status: 0, question: "CON LA V. Nombre dado a los miembros de los pueblos nórdicos originarios de Escandinavia, famosos por sus incursiones y pillajes en Europa"},
+    { letter: "w", answer: "sandwich", status: 0, question: "CONTIENE LA W. Emparedado hecho con dos rebanadas de pan entre las cuales se coloca jamón y queso"},
+    { letter: "x", answer: "botox", status: 0, question: "CONTIENE LA X. Toxina bacteriana utilizada en cirujía estética"},
+    { letter: "y", answer: "peyote", status: 0, question: "CONTIENE LA Y. Pequeño cáctus conocido por sus alcaloides psicoactivos utilizado de forma ritual y medicinal por indígenas americanos"},
+    { letter: "z", answer: "zen", status: 0, question: "CON LA Z. Escuela de budismo que busca la experiencia de la sabiduría más allá del discurso racional"},
+]
+	var name = window.prompt('Por favor introduce tu nombre');
+	window.alert(name + ' Bienvenido al juego del Pasapalabra. INSTRUCCIONS DEL JOC......');
+	var aciertos = 0;
+	var fallos = 0;
+	var points = 0;
+	var respostes = 0;
+	nuevaRonda();
+	function preguntes (){
+		for(var i=0; i<questions.length; i++){
+			
+			if(questions[i].status === 0){
+				var answer = window.prompt(questions[i].question);
+				if(answer ===questions[i].answer){
+					questions[i].status = 1;
+					points++;
+					respostes++;
+					aciertos++;
+					window.alert('Correcto! tienes ' + points + ' puntos.');
+				}else if(answer === 'pasapalabra'){
+					window.alert('Pasamos a la siguiente letra')
+				}else{
+					questions[i].status = 2;
+					points--;
+					fallos++;
+					respostes++;
+					window.alert('Incorrecto... Pierdes 1 punto.' + 'Tu puntuación actual es de ' + points + ' puntos.')
+				}
+			}else{}
+		}
+		if(respostes === questions.length){
+			window.alert('Ya has respondido a todas las preguntas... y tu puntuación es de ... ' + points + ' puntos!');	
+			showPoints();
+		}else{
+			window.alert('Vamos a por la siguiente ronda!')
+			nuevaRonda();
```

Comment:
do not use differents lenguages for variable declarations.

Reply:
Ok! I'll change it! Do you know if there is any way that when I select one word and I change it, I change all the same words in the file?